Makefile

```diff
@@ -1,18 +1,28 @@
 .PHONY: build-image std-install dep all install test api-reference FORCE
 
-all: jk
+PLUGINS =					\
+	plugins/jk-plugin-echo/jk-plugin-echo	\
+	$(NULL)
+
+all: jk $(PLUGINS)
 
 VERSION := $(shell git describe --tags)
 
-jk: pkg/__std/lib/assets_vfsdata.go FORCE
+ifneq ($(RW),yes)
```

Comment:
I'm guessing the motivation here is that occasionally it's a pain to have it not build because you've introduced a new dependency. You can just issue `go mod tidy` when that's the case.